remix-solidity/src/compiler/compiler.js

```diff
@@ -73,7 +73,8 @@ function Compiler (handleImportCall) {
   function onInternalCompilerLoaded () {
     if (worker === null) {
       var compiler
-      if (typeof (window) === 'undefined') {
+      var userAgent = navigator.userAgent.toLowerCase()
+      if (typeof (window) === 'undefined' || userAgent.indexOf(' electron/') > -1) {
```

Comment:
@iurimatias regarding the change we made recently, would this be ok to merge?
I believe that should be fine ...

Reply:
perhaps should we check for `Electron` too, or the lowered case `userAgent`.

https://developers.whatismybrowser.com/useragents/explore/software_name/electron-application/?order_by=-software_version

Reply:
Here is the issue where I found this solution https://github.com/electron/electron/issues/2288#issuecomment-337858978

I am not sure if we will need more checks :confused:

Reply:
Oh I see there is a nice little library for this https://github.com/cheton/is-electron, would you guys prefer to use that library instead ? @yann300 @iurimatias

Reply:
> would you guys prefer to use that library instead

not necessarily, i was just wondering whether we should also check with the `E` uppercase of electron.

Reply:
It is already transformed to lowercase `var userAgent = navigator.userAgent.toLowerCase()`

Reply:
ah ;) sorry i didn't see it